client/src/Features/PublishAssignment/PublishControlArea.tsx

```diff
@@ -5,56 +5,36 @@
 
 import React from 'react';
 import { IThemeOnlyProps, SimpleComponentStyles, IStylesOnly } from '../../Core/Utils/FluentUI/typings.fluent-ui';
-import { styled, MessageBarType, mergeStyles, AnimationClassNames } from '@fluentui/react';
+import { styled } from '@fluentui/react';
 import { themedClassNames } from '../../Core/Utils/FluentUI';
 import { PublishStatusIndicator } from './PublishStatusIndicator';
 import { PublishActionButtons } from './PublishActionButtons';
 import { useObserver } from 'mobx-react-lite';
-import { useStore } from '../../Stores/Core';
-import { AutohideMessageBar } from '../../Core/Components/AutohideMessageBar';
 
-export type PublishControlAreaStyles = SimpleComponentStyles<'root' | 'messageBar'>;
+export type PublishControlAreaStyles = SimpleComponentStyles<'root'>;
 
 const PublishControlAreaInner = ({ styles }: IStylesOnly<PublishControlAreaStyles>): JSX.Element => {
-  const assignmentStore = useStore('assignmentStore');
   const classes = themedClassNames(styles);
 
   return useObserver(() => (
+    <>

```

Comment:
any specific reason to add this fragment here?

Reply:
No, I added back with the other changes and forgot to remove this one! Rolling it off in the next commit.